Python/compile.c

```diff
@@ -1839,6 +1840,23 @@ compiler_add_yield_from(struct compiler *c)
     return 1;
 }
 
+static int
+compiler_pop_except_and_reraise(struct compiler *c)
+{
+    /* Stack contents
+     * [exc_info, lasti, exc]  ROT_THREE
+     * [exc, exc_info, lasti]  ROT_THREE
+     * [lasti, exc, exc_info]  POP_EXCEPT
+     * [lasti, exc]            RERAISE      1
```

Comment:
Nobody cares about performance of except* at the moment.

Reply:
Actually my reply here was incorrect - this does not impact only except*.

We can measure the performance, but if I ask "would we have merged POP_EXCEPT and RERAISE into a superinstruction?" the answer would probably be no because this is not a hot pair.

IIUC, @markshannon says that there is a cost to just having an unused opcode (he posted benchmark results showing that just having PREP_RERAISE_STAR and JUMP_IF_NOT_EG_MATCH slowed down benchmarks that were not using them).

Reply:
```
+------------------------+----------+-----------------------+
| Benchmark              | baseline | new                   |
+========================+==========+=======================+
| 2to3                   | 309 ms   | 319 ms: 1.03x slower  |
+------------------------+----------+-----------------------+
| chameleon              | 8.78 ms  | 8.65 ms: 1.01x faster |
+------------------------+----------+-----------------------+
| deltablue              | 4.96 ms  | 5.07 ms: 1.02x slower |
+------------------------+----------+-----------------------+
| fannkuch               | 421 ms   | 428 ms: 1.02x slower  |
+------------------------+----------+-----------------------+
| go                     | 163 ms   | 163 ms: 1.00x faster  |
+------------------------+----------+-----------------------+
| hexiom                 | 7.53 ms  | 7.40 ms: 1.02x faster |
+------------------------+----------+-----------------------+
| logging_silent         | 140 ns   | 134 ns: 1.05x faster  |
+------------------------+----------+-----------------------+
| logging_simple         | 7.03 us  | 7.10 us: 1.01x slower |
+------------------------+----------+-----------------------+
| mako                   | 13.1 ms  | 13.5 ms: 1.03x slower |
+------------------------+----------+-----------------------+
| nbody                  | 103 ms   | 108 ms: 1.04x slower  |
+------------------------+----------+-----------------------+
| nqueens                | 99.5 ms  | 102 ms: 1.03x slower  |
+------------------------+----------+-----------------------+
| pathlib                | 49.4 ms  | 48.8 ms: 1.01x faster |
+------------------------+----------+-----------------------+
| pickle                 | 11.2 us  | 11.3 us: 1.01x slower |
+------------------------+----------+-----------------------+
| pickle_dict            | 25.2 us  | 24.9 us: 1.01x faster |
+------------------------+----------+-----------------------+
| pickle_list            | 4.16 us  | 4.12 us: 1.01x faster |
+------------------------+----------+-----------------------+
| pickle_pure_python     | 388 us   | 393 us: 1.01x slower  |
+------------------------+----------+-----------------------+
| pidigits               | 193 ms   | 191 ms: 1.01x faster  |
+------------------------+----------+-----------------------+
| pyflate                | 524 ms   | 519 ms: 1.01x faster  |
+------------------------+----------+-----------------------+
| python_startup_no_site | 10.2 ms  | 10.1 ms: 1.01x faster |
+------------------------+----------+-----------------------+
| raytrace               | 369 ms   | 381 ms: 1.03x slower  |
+------------------------+----------+-----------------------+
| regex_dna              | 183 ms   | 180 ms: 1.02x faster  |
+------------------------+----------+-----------------------+
| regex_effbot           | 3.35 ms  | 3.38 ms: 1.01x slower |
+------------------------+----------+-----------------------+
| regex_v8               | 24.1 ms  | 24.3 ms: 1.01x slower |
+------------------------+----------+-----------------------+
| scimark_fft            | 391 ms   | 388 ms: 1.01x faster  |
+------------------------+----------+-----------------------+
| scimark_monte_carlo    | 74.4 ms  | 75.5 ms: 1.01x slower |
+------------------------+----------+-----------------------+
| scimark_sor            | 141 ms   | 144 ms: 1.02x slower  |
+------------------------+----------+-----------------------+
| spectral_norm          | 135 ms   | 136 ms: 1.01x slower  |
+------------------------+----------+-----------------------+
| sqlite_synth           | 2.76 us  | 2.79 us: 1.01x slower |
+------------------------+----------+-----------------------+
| unpickle               | 15.9 us  | 15.8 us: 1.01x faster |
+------------------------+----------+-----------------------+
| unpickle_list          | 4.78 us  | 4.84 us: 1.01x slower |
+------------------------+----------+-----------------------+
| xml_etree_process      | 68.0 ms  | 67.5 ms: 1.01x faster |
+------------------------+----------+-----------------------+
| Geometric mean         | (ref)    | 1.00x slower          |
+------------------------+----------+-----------------------+

Benchmark hidden because not significant (26): chaos, crypto_pyaes, django_template, dulwich_log, float, json_dumps, json_loads, logging_format, meteor_contest, python_startup, richards, scimark_lu, scimark_sparse_mat_mult, sqlalchemy_declarative, sqlalchemy_imperative, sympy_expand, sympy_integrate, sympy_sum, sympy_str, telco, tornado_http, unpack_sequence, unpickle_pure_python, xml_etree_parse, xml_etree_iterparse, xml_etree_generate
Ignored benchmarks (1) of ../cpython_baseline/baseline.json: regex_compile
```